ui/packages/tidb-dashboard-lib/src/apps/Statement/utils/useStatementTableController.ts

```diff
@@ -250,6 +268,12 @@ export default function useStatementTableController({
           list: res?.data || [],
           timeRange
         }
+        // if we have checked it is new backend (aka oldBackend === false)
+        // we don't check it again
+        if (oldBackend) {
+          // old backend api has no `summary_begin_time` field
+          setOldBackend(data.list[0]?.summary_begin_time === undefined)
+        }
```

Comment:
The `oldBackend` will be set to `true` when `data.list` is empty at the latest backend. Is that ok for the next round request?

Reply:
oh, nice catch! need to fix.

Reply:
and I am thinking to get new api or old api by checking api `/time_ranges` exist or not.

Reply:
> and I am thinking to get new api or old api by checking api /time_ranges exist or not.

Agree.